metis-authentication/metis-authentication-common/src/main/java/eu/europeana/metis/authentication/utils/MetisZohoOAuthPSQLHandler.java

```diff
@@ -39,17 +37,22 @@ public class MetisZohoOAuthPSQLHandler implements TokenStore {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MetisZohoOAuthPSQLHandler.class);
   private static final String USER_IDENTIFIER_STRING = "userIdentifier";
-  private static final SessionFactory sessionFactory;
+  private static SessionFactory sessionFactory;
   private static String clientId;
   private static String clientSecret;
 
-  static {
-    org.hibernate.cfg.Configuration configuration = new org.hibernate.cfg.Configuration();
-    configuration.addAnnotatedClass(MetisZohoOAuthToken.class);
-    configuration.configure();
-    ServiceRegistry serviceRegistry = new StandardServiceRegistryBuilder()
-        .applySettings(configuration.getProperties()).build();
-    sessionFactory = configuration.buildSessionFactory(serviceRegistry);
+  //  static {
```

Comment:
Are these comments needed? If not please remove
Otherwise please add a descriptive comment about this

Reply:
Indeed, leftovers I forgot to remove